src/app/locales/en/routes.yml

```diff
@@ -40,7 +40,7 @@ communities:
 community:
     menu:
         streams: Live Streams
-        info: Community Infos
```

Comment:
This change needs to be reverted.
The translation string is meant for the sub menu of an individual community. The "Community infos" route lists infos about a community, not about its streams.